src/main/scala/thepackage/payloads/responses/UserResponse.scala

```diff
@@ -0,0 +1,23 @@
+package thepackage.payloads.responses
+import java.time.OffsetDateTime
+import java.util.UUID
+
+import play.api.libs.json.Json
+import thepackage.models.User
+
+case class UserResponse(
+    id:        UUID,
```

Comment:
nit: could add transaction id uuid? for telemetry and testing later?

Reply:
Yeah probably something like that later.
I usually put transaction id/correlation id in headers so that it's easier to implement in the routing structure across all requests.